### ISSUE TITLE
pyright couldn't resolve submoudle.

### ISSUE THREAD

**Issue by JoeMob** [ayamir/nvimdots#562]
### Version confirmation

- [X] Confirm

### Following prerequisites

- [X] Confirm

### Neovim version

NVIM v0.8.2

### Operating system/version

win 11

### Terminal name/version

wezterm

### Branch info

main (Default/Latest)

### Fetch Preferences

SSH (use_ssh = true)

### Affected language servers

pyright

### How to reproduce the issue

1.Import open3d.
2.Call open3d's submodule visualization.

### Actual behavior

![image](https://user-images.githubusercontent.com/49267059/224475090-44103eb6-6f93-4e73-8cb0-76e7083dece1.png)

### Expected behavior

There should be no error reported.

### Support info

![image](https://user-images.githubusercontent.com/49267059/224475587-c98ef91d-69af-4ace-8a83-a41d2e685fb3.png)
![image](https://user-images.githubusercontent.com/49267059/224475597-9a32b8bf-8050-488e-a969-da03c6fd7ed0.png)

### Logs

```console
[START][2023-03-11 17:06:27] LSP logging initiated
[WARN][2023-03-11 17:06:27] ...lsp/handlers.lua:487	"stubPath typings is not a valid directory."
```

**Comment by ayamir** [comment]
We can't resolve this kind of problems because it belong to the inherent ability of `pyright`. All of my suggestion is to use other lsps.

**Comment by Jint-lzxy** [comment]
**TL;DR** If you are using a type stub for the library and you believe that the stub is missing a symbol, report the problem _(or submit a PR)_ with the maintainer of that type stub [(open3d)](https://github.com/isl-org/Open3D). We can't do much about this as it's server- and user-dependent.

***

This appears to be a problem with the type stub file that ships with `open3d`. It probably doesn't include the `visualization` symbol in it's top-level specification. It is up to library authors to determine what symbols they want to export as part of the official interface contract, and those symbols must be defined in the typestub file. `pyright` itself is very configurable and the default port without any configuration will generate many errors that might be insignificant. Please refer to https://microsoft.github.io/pyright/#/configuration.

A dirty workaround would be to simply import the submodule:
```python
import open3d.visualization
```

**Comment by JoeMob** [comment]
I disabled the error message, but pyright's behavior still doesn't match my expectation. Cuz open3d doesn't have stubs, `visualization` is just a subfolder under `open3d`'s root and it also imported in the `__init__.py` of `open3d.`
But my problems are not related to dotfiles, so I will close my issue.
